spark_pipeline_framework/transformers/framework_local_file_loader/v1/framework_local_file_loader.py

```diff
@@ -99,18 +99,18 @@ def _transform(self, df: DataFrame) -> DataFrame:
                 absolute_paths: List[str] = [filepath]
             else:
                 data_dir = Path(__file__).parent.parent.joinpath("./")
-                if isinstance(filepath, list):
-                    absolute_paths = [
-                        f"file://{data_dir.joinpath(path)}" for path in filepath
-                    ]
-                else:
-                    absolute_paths = [f"file://{data_dir.joinpath(filepath)}"]
+                absolute_paths = [f"file://{data_dir.joinpath(filepath)}"]
         elif isinstance(filepath, Path):
             data_dir = Path(__file__).parent.parent.joinpath("./")
             absolute_paths = [f"file://{data_dir.joinpath(filepath)}"]
         else:
             data_dir = Path(__file__).parent.parent.joinpath("./")
-            absolute_paths = [f"file://{data_dir.joinpath(path)}" for path in filepath]
+            absolute_paths = []
+            for path in filepath:
```

Comment:
Do we need some sort of `is(tuple, list)` check here?

Reply:
We didn't in the previous version, presumably because of the type check in the constructor, but I actually had the same thought when I was writing this. I've pushed an update that does a better job of checking.